src/PointwiseFunctions/AnalyticData/GrMhd/COCAL_reader/IO_input_CF_flco_export.f90

```diff
@@ -0,0 +1,22 @@
+subroutine IO_input_CF_flco_export(filenm,emd,ome,ber,radi)
```

Comment:
huh? What does this routine do? If for COCAL why not put it into a COCAL F90 module to avoid namespace pollution?

Reply:
ok, this seems to be a mostly full copy of COCAL. Why then did I provide a `COCAL_ROOT` above? Shouldn't all these files be removed?